### ISSUE TITLE
only player objects should be selectable in game mode

### ISSUE THREAD

**Issue by DaanMeijer** [OpenSAGE/OpenSAGE#317]
Currently, all GameObjects seem to be selectable, including civilian buildings and trees. The game should only allow for the LocalPlayers objects to be selectable

**Comment by Continous** [comment]
I think it should be noted that neutral structures and garrisonable structures need to be selectable.

**Comment by DaanMeijer** [comment]
I agree, this is about drag selection. I'll update the title accordingly.

**Comment by charliefoxtwo** [comment]
I was going to take this one, but it looks like it was fixed a while back in https://github.com/OpenSAGE/OpenSAGE/pull/318

Tested locally and the behavior seems to be as expected - drag selecting does not allow selection of non-local-player-owned unit.

Potentially a separate issue, however - selecting _or drag-selecting_ a _single_ enemy unit should show the unit or structure's information in the bottom right corner (the little picture and upgrades screen, whatever that's called) regardless of the owner.
